fees/incrementor_test.go

```diff
@@ -32,18 +32,21 @@ import (
 )
 
 func TestGasPriceIncrementor(t *testing.T) {
-	chid := int64(1)
+	chid := int64(9223372036854775790)
```

Comment:
Magic numbers?

Reply:
This is default value for unset chain id 🤷🏻

Reply:
I would still put it into config or at least into constant, so it would be more obvious what this magic number is about.